bin/presidium.js

```diff
@@ -42,13 +43,28 @@ var argv = yargs.usage('$0 command')
         presidium.watch(conf);
         presidium.serve(conf);
     })
+    .command('clean-gh-pages', 'Remove old versions.', function (yargs) {
```

Comment:
This command may be a little unclear for people...
I think a nice alternative is we add this to the existing `presidium clean` command. If a version is provided, delete that version: `npm run clean -v1.5`